app/views/static_pages/contributing/_contributing_options.html.erb

```diff
@@ -1,35 +1,26 @@
-<%
-  contributions = [
-    {
-      title: "Contribute to the Curriculum",
-      description: "The curriculum consists of the lessons and projects on this site. We are constantly expanding and updating the curriculum to keep it current and ensure it meets our students' needs in the ever changing world of web development. Our goal is to create the best free web development curriculum in the world, anyone is welcome to join in and help us in striving for this goal.",
-      path: "curriculum",
-    },
-    {
-      title: "Contribute to the Main Site",
-      description: "The site you are on right now is a Ruby on Rails application. We are always working on new features that improve the platform and help students in their learning journey. As with any Open Source project, there is also a constant backlog of outstanding issues and bugs that need to be fixed. Anyone is welcome to join in and start working on any of the issues we currently have posted in the Github repository for the site.",
-      path: "theodinproject",
-    }
-  ]
-%>
+<div>
```

Comment:
Previously the spacing between sections was pretty massive, originally I recreated it but I've gone through and given it a clean to try and rely more on the `page-containers` as you've suggested to get more consistent spacing. I think it looks better this way, but let me know if you think this has gone too far the other way

![image](https://user-images.githubusercontent.com/58506115/193796364-a884eaf6-29ac-4c55-b3ec-77911008b3f7.png)